scripts/test/runtests.sh

```diff
@@ -43,7 +46,8 @@ failCount=0
 
 for project in ${TestProjects[@]}
 do
-    ./corerun  "xunit.console.netcore.exe" "$project.dll" -xml "${project}-testResults.xml" -notrait category=failing
+    # corehost is on the PATH because it is in the stage 2 output
+    corehost "xunit.console.netcore.exe" "$project.dll" -xml "${project}-testResults.xml" -notrait category=failing
```

Comment:
Yes that's true it is in all stages, but we are hoping to re-use this script to test installer binaries post install. 
In that case (on Ubuntu, OSX) the binaries will be symlinked and corehost won't be on the path. 

At first we were renaming the output corehost in $TestBinRoot to "corehost" so we could use it like this, but it felt bad :smile: